src/main/java/org/openpnp/machine/reference/vision/ReferenceFiducialLocator.java

```diff
@@ -390,4 +604,84 @@ public Icon getPropertySheetHolderIcon() {
         // TODO Auto-generated method stub
         return null;
     }
+
+    // Get the fiducial pipeline from the config
+    public CvPipeline createPartPipeline(PartFiducialPipeline settings) {
+        CvPipeline cvp = null;
+        try {
+            if (settings != null) {
+                // try and get per-part pipeline
+                cvp = settings.getPipeline();
+                if (cvp == null) {
+                    // we tried to get it, but not there. Use default.
+                    try {
+                        cvp = defaultFiducialPipeline.clone();
+                    }
+                    catch (Exception e) {
+                    }
+                }
+            }
+            else {
+                // use default XML canned resource file
+                String xml = IOUtils.toString(ReferenceFiducialLocator.class
+                        .getResource("ReferenceFiducialLocator-DefaultPipeline.xml"));
+                cvp = new CvPipeline(xml);
+            }
+        }
+        catch (Exception e) {
+            throw new Error(e);
+        }
+        // try and set camera. If not in the GUI, this will fail and that's okay
+        try {
+            cvp.setCamera(Configuration.get().getMachine().getDefaultHead().getDefaultCamera());
+        }
+        catch (Exception e) {
+        }
+        return cvp;
+    }
+
+    @Root
+    public static class PartFiducialPipeline {
+        @Element(required = false)
+        protected CvPipeline pipeline;
+        
+        protected BufferedImage template;
+        protected double templateRotation;
+
+        public PartFiducialPipeline() {
+            pipeline = null;
+            template = null;
+            templateRotation = 99999.9D;
```

Comment:
What is the purpose of the 99999.9D here? Could it be changed for something more self evident? If not, can you add a comment explaining why it is the value that it is?